src/error.rs

```diff
@@ -63,7 +63,7 @@ pub enum Error {
     Url(#[from] UrlError),
     /// HTTP error.
     #[error("HTTP error: {}", .0.status())]
-    Http(Response<Option<String>>),
+    Http(Response<Vec<u8>>),
```

Comment:
Won't it make a bit more sense if we keep it as `Option` but with `Vec<u8>` inside?

This means that the code will be compatible with the previous semantics (if no body is set, it will be `None`), and also it would allow us to differentiate between the "empty body" and "no body".

What do you think?

Reply:
Agreed, will update this. :+1:

Reply:
It makes sense to switch the user side to `Option<Vec<u8>>` too, as the `server` and `client` modules currently use a common error type.